client/components/Dropdown.js

```diff
@@ -1,19 +1,60 @@
 import React from "react";
 import { Box, Center, Select, CheckIcon } from "native-base";
+import SelectDropdown from "react-native-select-dropdown";
+import { FontAwesome } from "@expo/vector-icons";
+import { StyleSheet } from "react-native";
 
 export const DropdownComponent = ({
   data,
   value,
   onValueChange,
+  icon,
+  iconPosition,
+  ItemStyle,
+  ItemTextStyle,
+  SelectTextStyle,
+  SelectStyle,
   placeholder,
   width,
   style,
 }) => {
-  // console.log("data in dropdown", data);
+  console.log("data in dropdown", style);
 
```

Comment:
if i `return null` here then the virtualized list error goes away FYI. So something here is still causing our virtualized list issue